src/Analysis/GWASV2/Components/SelectCohort/SelectCohortDropDown.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useQuery, queryConfig } from 'react-query';
+import { Select, Spin } from 'antd';
+import { fetchCohortDefinitions } from '../../Shared/cohortMiddlewareApi';
+import { useFetch } from '../../Shared/formHooks';
+import { useSourceContext } from '../../Shared/Source';
+
+const SelectCohortDropDown = ({
+  selectedCohort = undefined,
+  handleCohortSelect,
+}) => {
+  const { source } = useSourceContext();
+  const cohorts = useQuery(
+    ['cohortdefinitions', source],
+    () => fetchCohortDefinitions(source),
+    queryConfig,
+  );
+  const fetchedCohorts = useFetch(cohorts, 'cohort_definitions_and_stats');
+
+  const onChange = (selectedCohortDefinitionId) => {
+    console.log(`selected ${selectedCohortDefinitionId}`);
```

Comment:
I think you forgot to remove this console.log statement.

Reply:
done